tests/queries/0_stateless/01822_short_circuit.sql

```diff
@@ -153,3 +153,6 @@ select number % 2 and toLowCardinality(number) from numbers(5);
 select number % 2 or toLowCardinality(number) from numbers(5);
 select if(toLowCardinality(number) % 2, number, number + 1) from numbers(10);
 select multiIf(toLowCardinality(number) % 2, number, number + 1) from numbers(10);
+
+select 1 and greatest(number % 2, number % 3) from numbers(10);
```

Comment:
It's more convenient to always add new test files instead of appending to old ones.
\- more easy to separate test failures.